libs/common/src/key-management/crypto/services/encrypt.service.implementation.ts

```diff
@@ -211,42 +211,22 @@ export class EncryptServiceImplementation implements EncryptService {
     return result ?? null;
   }
 
-  async rsaEncrypt(data: Uint8Array, publicKey: Uint8Array): Promise<EncString> {
-    if (data == null) {
-      throw new Error("No data provided for encryption.");
-    }
-
-    if (publicKey == null) {
-      throw new Error("No public key provided for encryption.");
+  async encapsulateKeyUnsigned(key: SymmetricCryptoKey, publicKey: Uint8Array): Promise<EncString> {
```

Comment:
```suggestion
  async encapsulateKeyUnsigned(key: SymmetricCryptoKey, encapsulationKey: Uint8Array): Promise<EncString> {
```

Matching abstraction parameter names.